### DISCUSSION TITLE
Modifier+click to change step size in settings sidebar

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#460] [Q&A]
Each numeric stepper in the settings sidebar has a step value indicated by the panel. It would be nice if one could adjust the step size when clicking the arrows by holding down a modifier key.

For example, the Chrome devtools support this when changing values in the style inspector: https://developer.chrome.com/docs/devtools/shortcuts/

<img width="600" alt="image" src="https://user-images.githubusercontent.com/14237/185004031-a1955f26-4cca-42c1-8c86-8b80d31c4849.png">

**Comment by foxymiles** [comment]
This currently works for `shift` which gives you step * 10. Should we expand beyond that?

**Comment by jtbandes** [comment]
I think at least having a way to do smaller steps would be nice.

**Comment by foxymiles** [comment]
This would be very easy to add. We'd just need some consensus on what the key should be and how much it should reduce the step size.
